app/components/orders/order-summary.js

```diff
@@ -14,12 +14,13 @@ export default Component.extend(FormMixin, {
     );
   }),
 
+  discountCode: computed('data.discountCode', function() {
```

Comment:
What purpose does this computed statement server ? Couldn't you use `data.discountCode` directly ?